tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -191,6 +188,16 @@ func (s *stresser) Stress() error {
 	}
 	s.stressTable = createStressTable(stressEntries)
 
+	return nil
+}
+
+func (s *stresser) Stress() error {
+	if err := s.Setup(); err != nil {
```

Comment:
It should be used for initialize/re-initialize in the fields anytime you want. However, i see that it shouldn't part of the interface since initialize/re-initialize has nothing to do with stresser interface.